src/main/java/org/synyx/urlaubsverwaltung/absence/web/AbsenceOverviewViewController.java

```diff
@@ -272,7 +272,8 @@ private Map<LocalDate, PublicHoliday> getPublicHolidaysOfPerson(DateRange dateRa
             .entrySet().stream()
             .map(entry -> publicHolidaysService.getPublicHolidays(entry.getKey().getStartDate(), entry.getKey().getEndDate(), entry.getValue()))
             .flatMap(List::stream)
-            .collect(toMap(PublicHoliday::getDate, Function.identity()));
+            .collect(toMap(PublicHoliday::getDate, Function.identity(), (publicHoliday, publicHoliday2) -> new PublicHoliday(publicHoliday.getDate(),publicHoliday.getDayLength(), publicHoliday.getDescription().concat("/").concat(publicHoliday2.getDescription()))));
```

Comment:
Maybe we could move this code duplication to a static method or constructor method of `PublicHoliday`?

Reply:
I don't think so. I thought long about it and I think it is ok to return a list with all public holidays and the caller should decide how and what it will do with this information and I created an issue that will look deeper in it #4174